rest-api/api/base_api.py

```diff
@@ -1,19 +1,38 @@
+from datetime import datetime, timedelta
 import logging
 
 import app_util
 from flask import request, jsonify, url_for
 from flask_restful import Resource
 from model.utils import to_client_participant_id
+from model.requests_log import RequestsLog
 from dao.bigquery_sync_dao import deferred_bq_participant_summary_update
+from dao.base_dao import BaseDao
 from query import OrderBy, Query
 from werkzeug.exceptions import BadRequest, NotFound
 from google.appengine.ext import deferred
-
+from sqlalchemy import inspect
+from sqlalchemy.exc import NoInspectionAvailable
 
 DEFAULT_MAX_RESULTS = 100
 MAX_MAX_RESULTS = 10000
 
 
+def _deferred_save_raw_request(log):
```

Comment:
Seems like putting this in the base_dao would be the more natural place to put this instead of importing.

Reply:
I've moved the function to the base_dao.py file.